pallets/creditcoin/src/types/platform.rs

```diff
@@ -20,25 +22,63 @@ impl From<u64> for EvmChainId {
 }
 
 impl EvmChainId {
-	pub fn new(value: u64) -> Self {
+	pub const fn new(value: u64) -> Self {
```

Comment:
what is the benefit of `const` here?

Reply:
It allows this function to run at compile time, which makes
```const ETHEREUM: EvmChainId = EvmChainId::new(1);```
valid

Reply:
Got it!